Resources/AzureDevops/azure-pipelines.yml

```diff
@@ -10,55 +10,32 @@ resources:
   - repository: dfc-devops
     type: github
     name: SkillsFundingAgency/dfc-devops
-    ref: refs/tags/v1.6.4
+    ref: refs/tags/v1.6.12
     endpoint: 'GitHub (ESFA)'
 
 jobs:
 
 #Build and test resources
-- template: AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
-  parameters:
-    ArmTemplateRoot: $(ArmTemplateRoot)
+# - template: AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
```

Comment:
Leaving this commented this out will mean that the ARM templates won't get tested/pulled into the artifacts,  which will cause a release failure.

Please uncomment!